src/api/app/services/scm_status_reporter.rb

```diff
@@ -1,7 +1,9 @@
-class SCMStatusReporter
+class SCMStatusReporter < SCMExceptionHandler
   attr_accessor :event_payload, :event_subscription_payload, :scm_token, :state
 
   def initialize(event_payload, event_subscription_payload, scm_token, event_type = nil)
+    super()
```

Comment:
do you need the parenthesis here?

Reply:
Yes, otherwise it passes all parameters to the parent class. To avoid parameters I need paranthesis.